app/views/smart_proxies/index.html.erb

```diff
@@ -7,8 +7,13 @@
   <thead>
     <tr>
       <th class="col-md-2"><%= s_("SmartProxy|Name") %></th>
-      <th class="col-md-4"><%= s_("SmartProxy|Url") %></th>
-      <th class="col-md-4"><%= _("Features") %></th>
+      <% if SETTINGS[:locations_enabled] %>
+        <th class="col-md-2"><%= _('Locations') %></th>
+      <% end %>
+      <% if SETTINGS[:organizations_enabled] %>
+        <th class="col-md-2"><%= _('Organizations') %></th>
+      <% end %>
+      <th class="col-md-4"><%= _('Features') %></th>
       <th class="col-md-1"><%= _('Status') %></th>
       <th><%= _('Actions') %></th>
```

Comment:
Can you add hiding for smaller screens? In my case (Logs) I did this:

```html
      <tr>
          <th class="ca col-md-2 hidden-xs nbsp">Time</th>
          <th class="ca col-md-1 hidden-sm hidden-xs nbsp">Level</th>
          <th class="col-md-9 col-sm-10 col-xs-12 nbsp">Message</th>
          <th class="hidden"></th>
      </tr>
```

So level and then even timestamp disappears when you make the browser narrower.